### PR TITLE
Fix flake8 errors

### PR DESCRIPTION
**Pull request by rht**
In particular:
1. E501 (line too long, not all)
2. E231 (whitespace stuff)
3. F405 (import wildcard)
4. F841 (unused vars, this requires manual review)

### CODE REVIEW

File: quantecon/lss.py [hunk 1]
```diff
@@ -109,13 +109,16 @@ def __init__(self, A, C, G, H=None, mu_0=None, Sigma_0=None):
         # = Check Input Shapes = #
         ni, nj = self.A.shape
         if ni != nj:
-            raise ValueError("Matrix A (shape: %s) needs to be square" % (self.A.shape))
+            raise ValueError(
+                "Matrix A (shape: %s) needs to be square" % (self.A.shape))
         if ni != self.C.shape[0]:
-            raise ValueError("Matrix C (shape: %s) does not have compatible dimensions with A. It should be shape: %s" % (self.C.shape, (ni,1)))
+            raise ValueError("Matrix C (shape: %s) does not have compatible dimensions with A. It should be shape: %s" % (
+                self.C.shape, (ni, 1)))
```
Comment (mmcky) [inline quantecon/lss.py]: @rht is this splitting on the 80 char / line rule?

File: quantecon/optimize/tests/test_nelder_mead.py [hunk 2]
```diff
@@ -291,6 +290,7 @@ def test_goldstein_price(self):
         bounds = np.array([[-2., 2.],
                            [-2., 2.]])
 
+        # TODO: why is `bounds` not passed to the argument here?
```
Comment (mmcky) [inline quantecon/optimize/tests/test_nelder_mead.py]: @cc7768 would you have time to check this?

File: quantecon/tests/test_dle.py [hunk 3]
```diff
@@ -35,7 +35,7 @@ def setUp(self):
         deltah = np.array([[.9]])
         thetah = np.array([[1]]) - deltah
         ub = np.array([[30, 0, 0]])
-        x0 = np.array([[5, 150,1,0,0]]).T
+        # x0 = np.array([[5, 150, 1, 0, 0]]).T  # unused
```
Comment (mmcky) [inline quantecon/tests/test_dle.py]: @natashawatkins would you have time to check this?

File: quantecon/tests/test_kalman.py [hunk 4]
```diff
@@ -53,7 +53,7 @@ def test_stationarity(self):
 
 
     def test_update_using_stationary(self):
-        A, Q, G, R = self.A, self.Q, self.G, self.R
+        # A, Q, G, R = self.A, self.Q, self.G, self.R  # unused
```
Comment (mmcky) [inline quantecon/tests/test_kalman.py]: @natashawatkins would you have time to check this?

File: quantecon/tests/test_lqcontrol.py [hunk 5]
```diff
@@ -81,7 +81,7 @@ def test_stationary_mat(self):
         lq_mat = self.lq_mat
 
         f_answer = np.array([[-.95, -.95], [0., 0.]])
-        p_answer = np.array([[1., 0], [0., 0.]])
+        # p_answer = np.array([[1., 0], [0., 0.]])  # unused
```
Comment (mmcky) [inline quantecon/tests/test_lqcontrol.py]: @natashawatkins would you have time to check this?

**Comment by mmcky** [comment]
thanks @rht. 

I have tagged a few contributors to check a few of the F841 commented lines to make sure they aren't needed.

File: quantecon/game_theory/game_generators/tests/test_bimatrix_generators.py [hunk 6]
```diff
@@ -112,7 +112,7 @@ def test_seed(self):
     @raises(ValueError)
     def test_raises_value_error_too_large_inputs(self):
         n, k = 100, 50
-        g = tournament_game(n, k)
+        g = tournament_game(n, k)  # noqa: F841
```
Comment (oyamad) [inline quantecon/game_theory/game_generators/tests/test_bimatrix_generators.py]: I would rather remove "`g =`" (than adding "`# noqa: F841`"). (The same applies to other places.)

Comment (rht) [reply quantecon/game_theory/game_generators/tests/test_bimatrix_generators.py]: OK, will do that instead. I thought of adding `# noqa: F841` for consistency with the rest of the test cases.

**Comment by mmcky** [comment]
thanks for the updates @rht. 
@natashawatkins are you able to review the unused code items tagged in this PR?

File: quantecon/tests/test_dle.py (hunk 3 above)
Comment (natashawatkins) [reply quantecon/tests/test_dle.py]: it's not used anywhere else in this file, but I can't figure out why it's there in the first place

File: quantecon/tests/test_kalman.py (hunk 4 above)
Comment (natashawatkins) [reply quantecon/tests/test_kalman.py]: not used in this function - was probably just copied from above

File: quantecon/tests/test_lqcontrol.py (hunk 5 above)
Comment (natashawatkins) [reply quantecon/tests/test_lqcontrol.py]: doesn't appear to be used

**Comment by mmcky** [comment]
@rht would you mind deleting the unused items that are current commented?

**Comment by mmcky** [comment]
thanks for review @natashawatkins

**Comment by rht** [comment]
Assuming that that means all the unused variables detected by F841: I have just removed them.

**Comment by rht** [comment]
There are few remaining,
```
./docs/sphinxext/ipython_console_highlighting.py:62:9: F841 local variable 'tblexer' is assigned to but never used
./quantecon/robustlq.py:246:18: F841 local variable 'Q' is assigned to but never used
./quantecon/robustlq.py:246:21: F841 local variable 'R' is assigned to but never used
./quantecon/robustlq.py:247:9: F841 local variable 'beta' is assigned to but never used
./quantecon/robustlq.py:385:9: F841 local variable 'Q' is assigned to but never used
./quantecon/robustlq.py:385:12: F841 local variable 'R' is assigned to but never used
./quantecon/robustlq.py:395:9: F841 local variable 'sig' is assigned to but never used
./quantecon/kalman.py:162:12: F841 local variable 'k' is assigned to but never used
./quantecon/lss.py:395:18: F841 local variable 'H' is assigned to but never used
./quantecon/dle.py:80:9: F841 local variable 'phiing' is assigned to but never used
./quantecon/dle.py:309:9: F841 local variable 'Cc' is assigned to but never used
./quantecon/game_theory/repeated_game.py:125:5: F841 local variable 'C' is assigned to but never used
```

but I have to keep the PR lean for now.

**Comment by mmcky** [comment]
OK - thanks @rht

**Pull request merged**